client/src/stylesheets/app.sass

```diff
@@ -25,6 +25,7 @@ button
 button.btn-link.btn-plain
   padding: 0px
   color: white
+  background-color: black
```

Comment:
Just out of curiosity, Why do you need this modification?

Reply:
The mini scroll bar behind the transparent close button was bothering me.

Reply:
Makes sense.